MeshLib/convertMeshToGeo.cpp

```diff
@@ -63,12 +63,11 @@ bool convertMeshToGeo(const MeshLib::Mesh &mesh, GeoLib::GEOObjects &geo_objects
     const std::vector<MeshLib::Element*> &elements = mesh.getElements();
     const std::size_t nElems (mesh.getNumberOfElements());
 
-    auto materialIds = mesh.getProperties().getPropertyVector<int>(mat_name);
-    auto const materialIdExist = bool(materialIds);
+    auto const materialIds = mesh.getProperties().getPropertyVector<int>(mat_name);
 
     for (unsigned i=0; i<nElems; ++i)
     {
-        auto surfaceId = materialIdExist ? (*materialIds)[i]-bounds.first : 0;
+        auto surfaceId = !materialIds ? 0 : (*materialIds)[i] - bounds.first;
```

Comment:
i'm unsure if the line works in the same way as `auto surfaceId = !materialIds ? 0 : ((*materialIds)[i] - bounds.first);`